pages/common/git-pull.md

```diff
@@ -0,0 +1,11 @@
+# git pull
+
+> Fetch branch from a remote repo and merge it to current repo
+
+- Download the remote-tracking branch from default remote and merge it
```

Comment:
I don't think the term "remote-tracking branch" is self-explanatory. Maybe we can come up with a better way to put it?